### PR TITLE
Add Minify

### PR DESCRIPTION
**Pull request by johnfosborneiii**
#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates

#### For version bump PRs

- [X] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: minify.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      expected-commit: 65bbd3b7bcef06d21fac1af6a0ef7ff70a49800a
+
+  - uses: go/build
+    with:
+      packages: ./cmd/minify
+      ldflags: -s -w
+      output: minify
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/share/bash-completion/completions/
```
Comment (dlorenc) [inline minify.yaml:31]: If we install the bash completions we usually try to do it in a separate sub package.

File: minify.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      cp -p cmd/minify/bash_completion "${{targets.destdir}}"/usr/share/bash-completion/completions/minify
+      chmod 644 "${{targets.destdir}}"/usr/share/bash-completion/completions/minify
+      chmod 755 "${{targets.destdir}}"/usr/bin/minify
```
Comment (dlorenc) [inline minify.yaml:34]: You probably don't need to do this chmod here.

File: minify.yaml (hunk 1 above)
Comment (johnfosborneiii) [reply minify.yaml]: Fixed

File: minify.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  - uses: go/build
+    with:
+      packages: ./cmd/minify
+      ldflags: -s -w
+      output: minify
+
+  - runs: |
+      chmod 755 "${{targets.destdir}}"/usr/bin/minify
+
```
Comment (imjasonh) [inline minify.yaml:32]:
```suggestion

```

**Review by imjasonh — DISMISSED** [review]

**Review by dlorenc — APPROVED** [review]

**Pull request merged**